### ISSUE TITLE
Responsive Theme Versus Mobile Plugin

### ISSUE THREAD

**Issue by Elshara** [SocialEngine/phpv4-issues#474]
(Thanks for reporting an issue! Please make sure you click the link above to view the issue guidelines, then fill out the blanks below.)
## What are the steps to reproduce this issue?
1. … Will the mobile plugin be deprecated if a social engine default responsive theme rolls out?
2. … Will the mobile plugin have any more work done to it? Such as adding more pages for mobile compatibility in the layout editor besides the group, event and profile as well as custom home pages?
3. … Despite other third party mobile providers, the mobile plugin works very well. Will existing sites that use it be able to keep it?
## What happens?

… These are questions that probably would be addressed if the next version of Social Engine is to be the official 4.9.0 according to which a roadmap posted a year ago explains that a new responsive theme is planned.
## What were you expecting to happen?

… Either responsive theme fully replaces the mobile plugin, or the mobile plugin gets some work done to it. If the mobile plugin is deprecated, then the theme would disable it automatically if it is enabled. Otherwise, the mobile plugin be included anyways.
## Any logs, error output, etc?

(If it’s long, please paste to https://ghostbin.com/ and insert the link here.) 

```
Paste Log Here
```
## Any other comments?

… I love the browse menu in the mobile page view. Unfortunately that page isn't supported by the layout editor. I wish the desktop view had something similar because navigation can be quite long with several menus in the header and footer. Tab containers don't offer the same functionality if I was to recreate it.
## What versions of software are you using?

**Operating System:** … Windows 10

**SocialEngine PHP Version:** … 4.8.12

**Comment by Khushboo016** [comment]
Hi @Elshara ,

Regarding 1.
No, Mobi Plugin will not be be deprecated when SocialEngine's default responsive theme will roll out. Both Mobi plugin and the theme would play their significant roles. It will depend on a user that if he wants  to access the mobile site view in a mobile browser or if he chooses to access the Full site view of site in mobile browser. In first case Mobi plugin will be utilized and in second case responsive theme will be utilized for better responsiveness of the site.

Regarding 2.
Yes, surely we would consider adding more features to our Mobi plugin in future, but in the release 4.9 we would include only those things that we have already announced in the road-map of version 4.9.

Regarding 3. 
As we've already explained you above that Mobi plugin will not be deprecated at all, so the sites already using this plugin would be able to continue with it.

Thanks!
